monitoring.yaml

```diff
@@ -0,0 +1,12 @@
+user_name: "USERNAME"
+password: "AUTH"
```

Comment:
It's better to env for auth. Jenkins supports get credentials from console settings.

Reply:
This tool is not used for Jenkins.